src/ExpressionEvaluator/Core/Test/ResultProvider/ResultProviderTestBase.cs

```diff
@@ -115,13 +106,13 @@ internal bool HasUnderlyingString(object value)
         internal bool HasUnderlyingString(object value, Type type)
         {
             var clrValue = GetValueForUnderlyingString(value, type);
-            return clrValue.HasUnderlyingString();
+            return clrValue.HasUnderlyingString(inspectionContext: null);
```

Comment:
Ah, true, this is just a call to a test object. Seems like it might be useful to use DefaultInspectionContext, just so it's clear. Thanks.